src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,185 @@
+import React, {Component, createRef} from 'react';
+import uniqId from 'uniqid';
+import {Button} from '../Button/Button';
+import './AddUserForm.scss';
+import ImgWarning from './../../assets/icon-attention-png-clip-art.png'
+
+const CN = 'user-form';
+
+class AddUserForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentError: '',
+            inputErrorName: '',
+            inputErrorLastName: '',
+            errorInputName: '',
+            errorInputLastName: ''
+        };
+    }
+
+    // todo: добавить state, в него добавить пропертю, которая будет отображать есть ли ошибка или нет.
+    //  допустим это будет строка currentError
+    //  по умолчанию она будет пустой
+    //  если произошла ошибка - в эту пропертю запишется строка, какая именно ошибка произошла
+
+    userNameRef = createRef();
+    userLastNameRef = createRef();
+
+    onInputChange = (e) => {
+
+        if (e.target.value) {
+            this.setState({
+                currentError: ''
+            })
+        }
+
+        if (this.userNameRef.current.value) {
+            this.setState({
+                inputErrorName: '',
+                errorInputName: ''
+            })
+        }
+
+        if (this.userLastNameRef.current.value) {
+            this.setState({
+                inputErrorLastName: '',
+                errorInputLastName: ''
+            })
+        }
+
+        // todo: если в стейте была записана какая-то ошибка - зачищаем ее при изменении инпута
+        //  поскольку в ситуации если пользователь ничего - не ввел - увидел ошибку - понял, что не так - начал вводить данные
+        //  было бы хорошо не пугать его висящим сообщением, что все таки что-то не так
+    };
+
+    onSubmit = (e) => {
+        e.preventDefault();
+
+        if (this.userNameRef.current.value.trim() === '' && this.userLastNameRef.current.value.trim() === '') {
+            this.setState({
+                currentError:
+                    <div className='warning'>
+                        <div>
+                            <h2>WARNING!!! WRITE SOMETHING!!!</h2>
+                            <img src={ImgWarning} alt="Warning"/>
+                        </div>
+                    </div>,
```

Comment:
You shouldn't set html or jsx inside state. Instead you can put simple string

`````
 currentError: 'WARNING!!! WRITE SOMETHING!!!'
````
But in render function actually use this jsx 

there would be something like:
`
const {currentError} = this.state;
....
{
    !!currentError && ( <div className='warning'>
                        <div>
                            <h2>{currentError}</h2>
                            <img src={ImgWarning} alt="Warning"/>
                        </div>
                    </div>)
}
`